src/components/menu/menu.ts

```diff
@@ -22,10 +26,34 @@ import {
 })
 export class MdMenu {
   private _showClickCatcher: boolean = false;
+  private _classList: Object;
```

Comment:
`Object` isn't super clear; add a comment explaining that this is the input into `ngClass` with an example?